migrations/20230311081917_envelopes_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS envelopes (
+  id              INTEGER PRIMARY KEY AUTOINCREMENT,
+  own_key         TEXT,
+  sampling_key    TEXT,
```

Comment:
Can we make these fixed width 32 byte arrays? That should make the lookup faster (at least in theory).